include/libopencm3/stm32/f7/dma2d.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2016 Chuck McManis <cmcmanis@mcmanis.com>
```

Comment:
same as elsewhere.

Reply:
should i just remove the whole header then? or just the copyright?